ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -109,6 +109,10 @@ open class PillButton: UIButton {
         if #available(iOS 15.0, *) {
             var configuration = UIButton.Configuration.plain()
             configuration.attributedTitle = AttributedString(pillBarItem.title)
+
+            // Workaround for Apple bug: when UIButton.Configuration is used with UIControl's isSelected = true, accessibilityLabel doesn't get set automatically
+            accessibilityLabel = pillBarItem.title
```

Comment:
This is only called in init, right? So if we change the title later this won't be updated? Unfortunately that is an actual scenario in our usage of PillButton.

Reply:
well.. this version of the PR doesn't have lukas change that actually makes the pillbutton title mutable.